### PR TITLE
Make the notification icon black when its background color is light

### LINKED ISSUE #13234: The notification icon remains white even if the color of the status bar is white or other light one
###### Reproduction Steps

1. Show the notification icon on the status bar whose color is white or other light one

###### Expected Result
As with other icons, the AnkiDroid notification icon becomes black

###### Actual Result
In contrast to the other icons, the AnkiDroid notification icon remains white

Record_2023-02-11-11-12-21_c428a0327e1e5e78cdd71c6b04816edf_AdobeExpress (1)
image

###### Debug info
AnkiDroid Version = 2.16alpha93

Android Version = 11

Manufacturer = OPPO

Model = CPH2013

Hardware = qcom

Webview User Agent = Mozilla/5.0 (Linux; Android 11; CPH2013 Build/RKQ1.200903.002; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/109.0.5414.85 Mobile Safari/537.36

ACRA UUID = 4425b0ff-647b-4f6a-b482-8e90da53bd7c

New schema = true

Scheduler = std3

Crash Reports Enabled = true

DatabaseV2 Enabled = true

###### Research

- [x] I have read the [support page](https://ankidroid.org/docs/help.html) and am reporting a bug or enhancement request specific to AnkiDroid
- [x] I have checked the [manual](https://ankidroid.org/docs/manual.html) and the [FAQ](https://github.com/ankidroid/Anki-Android/wiki/FAQ) and could not find a solution to my issue
- [x] I have searched for similar existing issues here and on the user forum
- [x] (Optional) I have confirmed the issue is not resolved in the latest alpha release ([instructions](https://docs.ankidroid.org/manual.html#betaTesting))

### PR DESCRIPTION
**Pull request by snowtimeglass**
## Purpose / Description
The notification icon remains white even if the color of the status bar is white or other light one.
Record_2023-02-11-11-12-21_c428a0327e1e5e78cdd71c6b04816edf_AdobeExpress (1)
image
This commit makes the icon black in such background.

## Fixes
Fixes #13234

## Approach
Make new notification icon files from the current one (`drawable/ic_star_notify.xml`) with Asset Studio (`File` -> `New` -> `Image Asset`)
image
image
By the process, the original file, `drawable/ic_star_notify.xml`, is deleted automatically. 

## How Has This Been Tested?
Checked on a physical device 
Android Version: 11
Compile SDK Version: 33
VID_20230211134539_AdobeExpress
image

## Checklist

- [x] You have a descriptive commit message with a short title (first line, max 50 chars).
- [ ] ~~You have commented your code, particularly in hard-to-understand areas~~
- [x] You have performed a self-review of your own code
- [x] UI changes: include screenshots of all affected screens (in particular showing any new or changed strings)
- [ ] ~~UI Changes: You have tested your change using the [Google Accessibility Scanner](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor)~~

### CODE REVIEW

**Review by david-allison — COMMENTED** [review]
Do you have documentation that this is the correct way of creating a dark icon?

I agree with the outcome (thank you!!), but I would strongly suspect there's a way to do this using less space via the XML. 

Typically Android pushes us towards XML rather than using .PNGs

**Comment by snowtimeglass** [comment]
> Do you have documentation that this is the correct way of creating a dark icon?

No, I don't.
 
> I would strongly suspect there's a way to do this using less space via the XML.


Actually, I have suspected that, too.

At first, I just rewritten the original icon file (`drawable/ic_star_notify.xml`) with the new XML file (and deleted `drawable-anydpi-v24` folder and 4 PNG files, which had newly been added by Asset Studio). The single replacement makes the outcome above properly in my device.

I was about to commit the single change, but I hesitated it because I didn't have the knowledge to justify deleting the `drawable-anydpi-v24` folder and 4 PNG files. In the end, I committed the changes as Asset Studio did.

> Typically Android pushes us towards XML rather than using .PNGs

I agree. I am positive about deleting the PNG files if you think it is appropriate.

**Comment by david-allison** [comment]
I'd prefer the single change and will take responsibility if it's the wrong move. Could you rebase it in and I'll get it reviewed

Thanks for the reply!

**Comment by snowtimeglass** [comment]
I've rebased and force-pushed it. May these procedures be appropriate.
I've also confirmed that the outcome above is reproduced after this commit.

File: AnkiDroid/src/main/res/drawable/ic_star_notify.xml [hunk 1]
```diff
@@ -1,13 +1,18 @@
...
-      android:fillColor="#ffffff"
-      android:strokeWidth="1"/>
+    android:viewportHeight="124"
+    android:tint="#FFFFFF"
```
Comment (david-allison) [inline AnkiDroid/src/main/res/drawable/ic_star_notify.xml:6]: I'd have thought that this would have forced `FFFFFF`. Learning experience. Thanks!!

**Review by david-allison — APPROVED** [review]

**Review by BrayanDSO — APPROVED** [review]
Thanks!

**Pull request merged**